package.json

```diff
@@ -21,7 +21,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@workos-inc/node": "^6.8.0",
+    "@workos-inc/node": "^7.12.0",
```

Comment:
Since we're updating to v7, we should add the `appInfo` object when initializing the WorkOS client. That can go in the version bump PR though.